zulip.md

```diff
@@ -0,0 +1,61 @@
+
+##   Learning Resources :eyeglasses:
+### To keep this community safe for you and others.
+
+1.  [Code of Conduct](/Contributing/CODE_OF_CONDUCT.md)
+
+2.  [Reporting Guidelines](/Contributing/REPORTING_GUIDELINES.md) 
+
+3.  [Zulip official documentation on how to getting started](https://zulipchat.com/help/getting-started-with-zulip)  
+
+##  Streams :point_down:
+    
+1. Category baseds: These are streams based on a specific type of contribution (e.g.: quality assurance, coding, documentation, etc). Join streams of your interest such as:
+
+
+    1.  [#coding](https://anitab-org.zulipchat.com/#narrow/stream/216321-coding) :computer:
+    2.  [#design](https://anitab-org.zulipchat.com/#narrow/stream/216323-design) :art:
+    3.  [#documentation](https://anitab-org.zulipchat.com/#narrow/stream/216326-documentation) :notebook_with_decorative_cover:    
+    4.  [#outreach-research](https://anitab-org.zulipchat.com/#narrow/stream/216324-outreach-research) :microscope:
+    5.  [#quality-assurance](https://anitab-org.zulipchat.com/#narrow/stream/216325-quality-assurance) :heavy_check_mark:
+
+
+2. Project Specific: You can join also project-specific streams, such as: 
+    *   [#mentorship-system](https://anitab-org.zulipchat.com/#narrow/stream/222534-mentorship-system) :mortar_board:
+        *   [anitab-org/mentorship-backend](https://github.com/anitab-org/mentorship-backend)
+        *   [anitab-org/mentorship-flutter](https://github.com/anitab-org/mentorship-flutter)
+        *   [anitab-org/mentorship-android](https://github.com/anitab-org/mentorship-android)
+        *   [anitab-org/mentorship-ios](https://github.com/anitab-org/mentorship-ios)
+
+    *   [#open-source-progs](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs) 
+        *   [anitab-org/open-source-programs-web](https://github.com/anitab-org/open-source-programs-web)
+        *   [anitab-org/open-source-programs-backend](https://github.com/anitab-org/open-source-programs-backend)
+        
+    *   [#portal](https://anitab-org.zulipchat.com/#narrow/stream/222540-portal) 
+        *   [anitab-org/portal](https://github.com/anitab-org/portal)
+
+    *   [#stem-diverse-tv](https://anitab-org.zulipchat.com/#narrow/stream/225705-STEM-diverse-tv)        
+        *   [anitab-org/stem-diverse-tv](https://github.com/anitab-org/stem-diverse-tv)
+        
+    *  [#vms](https://anitab-org.zulipchat.com/#narrow/stream/222539-vms)                
+        *   [anitab-org/vms](https://github.com/anitab-org/vms)
+
+    *   [#anitab-org.github.io](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio)                
+        *   [anitab-org/anitab-org.github.io](https://github.com/anitab-org/anitab-org.github.io)
+        
```

Comment:
Good job @kakabisht , very easy to read!
You can add the following, which is one of the new AnitaB.org projects from GSoC:
* [#bridge-in-tech](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech)
   * [anitab-org/bridge-in-tech-backend](https://github.com/anitab-org/bridge-in-tech-backend)
   * [anitab-org/bridge-in-tech-web](https://github.com/anitab-org/bridge-in-tech-web)

Reply:
HI, @foongminwong fixed the the request, let me know if i could further modify something.